### DISCUSSION TITLE
Customizable self timer function.

### DISCUSSION THREAD

**Discussion by Tylerdillonphoto** [gopro/labs#825] [Feature Requests]
Maybe I missed it when I was tinkering with things but I'd like a way to set the self timer function in photo/burst mode to any number of seconds. And one greater than 10 seconds.

My use case is shooting drift cars. I can trigger the remote as they leave the line and know that it takes about 'x' number of seconds to reach a particular part of the course, (which would be out of range of the remote) and let the burst mode handle the rest.

So they leave the line. Get to the corner I want to shoot about 45 seconds later. And I can set the burst to do 6 or 10 seconds or whatever and get a handful of RAW files to work with.

**Comment by dnewman-gpsw** [comment]
Maybe possible with experimentation. However the stock timer is not configurable through Labs, so you want have to script a delay behavior with Blockly https://gopro.github.io/labs/build/
